package.json

```
@@ -44,6 +46,7 @@
     "eslint-plugin-header": "3.0.0",
     "eslint-plugin-import": "2.21.1",
     "fs-extra": "9.0.1",
+    "html-differ": "^1.4.0",
```

```suggestion
    "html-differ": "1.4.0",
```
Use pinned dependencies

---

hey @rofe I think you need to replace the test:pages in package.json; step where TEST_DOMAIN=$TEST_DOMAIN to hlx.page! This is done in the other tests; or set the process.env.TEST_DOMAIN!

---

Thanks for the hint, the test is passing now!